src/applications/edu-benefits/complaint-tool/reducers/schoolSelect.js

```diff
@@ -61,8 +64,9 @@ export default function schoolSearch(state = initialState, action) {
         return { city, country, facilityCode, name, state: institutionState, zip };
       });
       const pagesCount = Math.ceil(searchResultsCount / 10);
-      const showInstitutions = true;
+      const showInstitutions = institutions.length > 0;
       const showInstitutionsLoading = false;
+      const showNoResultsFound = institutions.length < 1;
```

Comment:
Driving this with the data that's parsed from the institutions attribute rather than the `meta.count` attribute.